Onepunch/README.md

```diff
@@ -1,26 +1,37 @@
 # Onepunch
+
 ## Screenshots
-![Home](./__Home.png)
-![Profile](./__Profile.png)
-![Playlist](./__Playlist.png)
+
+![Home_legacy](./__Home.png)
+
+<table>
+  <tr>
+    <td><img src="./__Playlist.png" alt="2" width = 360px> Playlist_legacy </td>
+    <td><img src="./__Profile.png" alt="3" width = 360px> Profile_legacy </td>
+  </tr>
+</table>
 
 ## More
 
 #### About
-A simple gruvified spotify theme. 
 
-#### First Aid Kit  ¯\\_(ツ)\_/¯
+A simple gruvified spotify theme.
+
+#### First Aid Kit ¯\\\_(ツ)\_/¯
 
-- To change color schemes **`spicetify config color_scheme color_scheme_name`** (change the **`color_scheme_name`** to dark/light)
-- Tested on macOS only and pretty much everything worked. ﾍ(･_|
+- **The screenshots here are from the old legacy UI.**
+- To change color schemes:
+  ```shell
+  spicetify config color_scheme [wal/dark/light]
```

Comment:
Where's the light theme?

Reply:
As said in other comments, I suggest you to remove `wal` as color scheme

Reply:
light theme is there. I haven't edited it that's why maybe it's not showing here.

yeah I'll remove it,  add new screenshots and then update this pull request. Thanks :)